### PR TITLE
Allow patrons to add a new account directly from the account picker dialog.

### PR DESCRIPTION
**Pull request by twaddington**
**What's this do?**
Adds an 'Add new account' option to the account switcher and makes the currently active account more obvious.

**Why are we doing this? (w/ JIRA link if applicable)**
Improved Patron experience.

**How should this be tested? / Do these changes have associated tests?**
1. Tap the account icon in the upper left of the catalog.
2. Verify that the 'Add new account' option is shown at the bottom of the list.
3. Tap 'Add new account' and verify the account registry fragment is displayed, as expected.

**Dependencies for merging? Releasing to production?**
n/a

**Has the application documentation been updated for these changes?**
n/a

**Did someone actually run this code to verify it works?**
@twaddington

**Screens**
- [Screenshot_1595529410](https://user-images.githubusercontent.com/107117/88324631-d4001980-ccd8-11ea-90e5-ca207952a5c7.png)

### CODE REVIEW

**Comment by io7m** [comment]
This all looks good to me. One thing though: To be LFA-friendly, we need to respect the value of `SettingsConfigurationServiceType.allowAccountsAccess`. That is, if that property is `false`, users shouldn't be allowed to add or delete accounts. Normally this flag just hides access to the accounts section in the settings screen, but if they can now add accounts via the picker, then it needs to be handled there too.

**Comment by twaddington** [comment]
> To be LFA-friendly, we need to respect the value of `SettingsConfigurationServiceType.allowAccountsAccess`. That is, if that property is `false`, users shouldn't be allowed to add or delete accounts. Normally this flag just hides access to the accounts section in the settings screen, but if they can now add accounts via the picker, then it needs to be handled there too. &mdash; @io7m 

Excellent feedback Mark, thanks. I'll make that change.

**Comment by twaddington** [comment]
@io7m updated!

**Comment by io7m** [comment]
There's one outstanding issue with this, but we don't need to correct it right away: The `ui-settings` module shouldn't depend on the `ui-catalog` module; it's bad form for a large implementation module to depend on another large implementation module. The problem here is that the `allowAccountsAccess` flag is only accessible via a service who's API is exported from the `ui-settings` module. What we need to do is move all of those configuration services into the `buildconfig` API.

Go ahead and merge this. I'll submit a PR for the buildconfig change afterwards.

File: simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountRegistryFragment.kt [hunk 1]
```diff
@@ -248,7 +248,7 @@ class AccountRegistryFragment : Fragment() {
     if (host is ToolbarHostType) {
       host.toolbarClearMenu()
       host.toolbarSetTitleSubtitle(
-        title = this.requireContext().getString(R.string.accounts),
+        title = this.requireContext().getString(R.string.accountAdd),
```
Comment (MalcolmMcFly) [inline simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountRegistryFragment.kt:251]: `R.string.account_add` ?

**Comment by twaddington** [comment]
> Everything looks good to me. I didn't realize that we were camel-casing string resource names. If the project uses that convention then you can ignore that comment.

Yeah, all the string names in the `simplified-ui-accounts` module are camel cased.

**Pull request merged**